Resources/Prototypes/Floof/Entities/Mobs/Cyborgs/quadborg.yml

```
@@ -116,4 +119,143 @@
   - type: AccessReader
     access: [["Cargo"], ["Salvage"], ["Command"], ["Research"]]
   - type: SiliconLawProvider
-    laws: Corporate
\ No newline at end of file
+    laws: Corporate
+
+- type: entity
+  id: BorgChassisQuadserv
+  parent: BorgChassisQuad
+  components:
+  - type: Sprite
+    sprite: Floof/Mobs/Silicon/quad.rsi
+    scale: 0.7, 0.7
+    layers:
+    - state: quadspriteserv
+      map: ["movement"]
+    - state: quadspriteservl
+      map: ["enum.BorgVisualLayers.Light"]
+      shader: unshaded
+      visible: false
+    - state: quadspriteservr
+      shader: unshaded
+      map: ["light"]
+      visible: false
+  - type: IntrinsicRadioTransmitter
+    channels:
+    - Service
+    - Binary
+    - Common
+    - Science
+  - type: ActiveRadio
+    channels:
+    - Service
+    - Binary
+    - Common
+    - Science
+  - type: BorgChassis
+    maxModules: 6
```

```suggestion
    maxModules: 4
```
Normal Service Cyborgs has only 4, 6 is only for Generic Type Borgs.

---

the service is functioning as the generic for quad borgs due to the number of available modules

---

Then make it only a generic borg or due to the following changes: Check current [Service Cyborg](https://github.com/Fansana/floofstation1/blob/master/Resources/Prototypes/Entities/Mobs/Cyborgs/borg_chassis.yml) and there you can see the normal ammount of modules required.

---

negative, Service is the generic of Quadborgs. that's an intended mechanic.

---

Then remove BorgModuleService, were not gona boost Quad to be "better" then the original.

---

it's not better, it's different. the "generic" quad is security which only gets 4 slots 

instead, seeing as I'm sure you don't want security to have 6 slots, I put it on service. 

unless you would rather I give security 6 instead.

---

Then make a generic variant.

---

there is one, it's called security. which would you rather have the 6 slots